devdocs/settingUp.md

```diff
@@ -22,10 +31,10 @@ Important: When a version is specified, please install that version instead of t
    Further instructions for installing can be found at https://developers.google.com/eclipse/docs/using_sdks.
 8. Install the latest [TestNG Eclipse plugin](http://testng.org/doc/download.html).
 
-##Setting up the dev server
+##<a name="setDevServer"></a>Setting up the dev server
```

Comment:
See if you can avoid adding anchors. I think markdown supports linking to headings without adding anchor tags.